### ISSUE TITLE
[Theme request] JetBrains Darcula (p.s. not Dracula*)

### ISSUE THREAD

**Issue by ianpan870102** [hlissner/emacs-doom-themes#429]
Not sure if this is the way to propose new themes -- but I thought JetBrain's Darcula theme would be a nice addition to make this comprehensive theme pack even more perfect, as it's classic and seen as default in PyCharm, IDEA, CLion etc.

**Comment by ianpan870102** [comment]
I've just created a port for private use here: https://github.com/ianpan870102/jetbrains-darcula-emacs-theme
I'd love it if someone could port it to Doom Themes using my code base.

**Comment by ema2159** [comment]
Awesome! That helps a lot

**Comment by asdasfasf121** [comment]
yeah man i totally get it, deepseek rate limits can be super frustrating af.

i was actually working on a side project a while back to deal with this exact issue - i built an open-source proxy/pool that aggregates deepseek keys to make sure you can always get the keys you need. yeah it's not just some fancy solution, it's really helped me out when i need to switch between projects quickly. 

as for trying it out, i've got a few extra free trial keys floating around if you want to test it out. no guarantees it'll blow your mind but it's been working solidly for me so far.

check it out if you're interested: https://lucky.wangzhonghao.me
